src/composables/invites.ts

```diff
@@ -0,0 +1,93 @@
+import { STORAGE_KEYS } from '@/constants';
+import {
+  AE_AMOUNT_FORMATS,
+  AeSdk,
+  Encoded,
+  Node,
+} from '@aeternity/aepp-sdk';
+import type { IInvite } from '@/types';
+import { tg } from '@/store/plugins/languages';
+import { getAccountFromSecret } from '@/protocols/aeternity/helpers';
+import migrateInvitesVuexToComposable from '@/migrations/006-invites-vuex-to-composable';
+import { useStorageRef } from './composablesHelpers';
+import { useModals } from './modals';
+import { useNetworks } from './networks';
+
+const invites = useStorageRef<IInvite[]>(
+  [],
+  STORAGE_KEYS.invites,
+  {
+    migrations: [
+      migrateInvitesVuexToComposable,
+    ],
+  },
+);
+
+export function useInvites() {
+  const { activeNetwork } = useNetworks();
+  const { openDefaultModal } = useModals();
+
+  function addInvite(secretKey: Buffer) {
+    invites.value.unshift({
+      secretKey: secretKey.toJSON(),
+      createdAt: Date.now(),
+    });
+  }
+
+  function removeInvite(secretKey: Buffer) {
+    invites.value = invites.value.filter((invite) => invite.secretKey !== secretKey);
+  }
+
+  async function claimInvite({
+    secretKey,
+    recipientId,
+    amount = '0',
+    isMax = false,
+  }: { secretKey: Buffer; recipientId: Encoded.AccountAddress; amount?: string; isMax: boolean }) {
+    const aeSdk = new AeSdk({
+      nodes: [{
+        name: activeNetwork.value.name,
+        instance: new Node(activeNetwork.value.protocols.aeternity.nodeUrl),
+      }],
+      accounts: [getAccountFromSecret(secretKey)],
+    });
+    if (!isMax) {
+      await aeSdk.spend(
+        amount,
+        recipientId,
+        // @ts-ignore
+        { denomination: AE_AMOUNT_FORMATS.AE },
+      );
+    } else {
+      await aeSdk.transferFunds(
+        1,
```

Comment:
Maybe it's worth to comment this line why are we sending `1`

Reply:
@CedrikNikita can you provide some info about this?

Reply:
First argument is a percentage from 0 to 1, where 0 is 0% and 1 is 100%.